### PR TITLE
Add conditional tracking pixel

### PR DESCRIPTION
**Pull request by eric-schneider**
**JIRA:** [DOC-4013](https://datastax.jira.com/browse/DOC-4013)
**Resolves**: https://github.com/riptano/docs-ui/issues/117

This PR adds a new partial, `tracking-pixel.hbs`, that implements a conditional tracking pixel with a variable ID. The tracking pixel code will only get built into pages when the `site.keys.scarf_tracking_pixel` key is defined in the playbook and assigned a value:

```yaml
site:
  keys:
    scarf_tracking_pixel: 'PIXEL_ID'
```

See related [playbook change](https://github.com/riptano/datastax-docs-site/blob/fb02f927aa13c002db7e8c0677f8c9e0bb250d96/antora-playbooks/release-unified.yaml#L20).

From what I understand, tracking pixels are usually placed right before the closing `</body>` tag so that they don’t delay the loading of any visible content. For this reason, I've inserted the `tracking-pixel` partial at the bottom of `body.hbs`. 

@colegoldsmith, this ensures that the pixel will get included on every page, correct?

I've also seen it suggested that it's a good idea to hide the pixel with a CSS style so that the pixel doesn’t interact with other CSS on the page. For this reason, I've added `style="display:none` to the pixel.

[DOC-4013]: https://datastax.jira.com/browse/DOC-4013?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/partials/body.hbs [hunk 1]
```diff
@@ -11,3 +11,4 @@
   </div>
 </div>
 {{> body-end-scripts}}
+{{> tracking-pixel}}
```
[the remark this answers is not in the document]
Comment (eric-schneider) [reply src/partials/body.hbs:14]: Funny enough, that was my initial implementation, but I got cold feet when I saw that the `intercom`partial was also a script. But I think you're right that it probably makes sense to treat `body-end-scripts` as the "everything that goes before `</body>`" partial.

@colegoldsmith what do you think?

[the remark this answers is not in the document]
Comment (eric-schneider) [reply src/partials/body.hbs:14]: Good ideas 👍 I'll make sure to include these updates in my next commit.

[the remark this answers is not in the document]
Comment (colegoldsmith) [reply src/partials/body.hbs:14]: I agree you could put this within body-end-scripts instead of body

**Review by colegoldsmith — APPROVED** [review]
From the discussion I would suggest moving to body-end-scripts but it's not a huge deal. Looks good otherwise!

**Pull request merged**